dGame/dComponents/InventoryComponent.h

```diff
@@ -379,7 +379,7 @@ class InventoryComponent final : public Component {
 	 * All the inventory this entity possesses
 	 */
 	std::map<eInventoryType, Inventory*> m_Inventories;
-
+	std::vector<LWOOBJID> invTransferred;
```

Comment:
The way you designed this logic I would recommend just using a queue instead of a vector.  Since you are choosing to let stacking items fill in the previous slot without overriding its age, a vector no longer makes sense as a container to store buyback order in.  Also a better name here would be buybackItems.  There are more than 1 transfer inventory and this name could be confusing.